Demo/Application/Features/ShopperInsightsViewController.swift

```diff
@@ -9,7 +9,7 @@ class ShopperInsightsViewController: PaymentButtonBaseViewController {
     lazy var shopperInsightsClient = BTShopperInsightsClient(apiClient: apiClient)
     lazy var payPalClient = BTPayPalClient(apiClient: apiClient)
     lazy var venmoClient = BTVenmoClient(
-        apiClient: apiClient,
+        authorization: "sandbox_9dbg82cq_dcpspy2brwdjr3qn",
```

Comment:
I don't think we want to hardcode this string. Should be able to use `authorization` from `PaymentButtonBaseViewController`